library/oneview_storage_volume_attachment.py

```diff
@@ -16,7 +16,13 @@
 ###
 
 from ansible.module_utils.basic import *
-from hpOneView.oneview_client import OneViewClient
+
+try:
+    from hpOneView.oneview_client import OneViewClient
+    from hpOneView.exceptions import HPOneViewException
+    from hpOneView.exceptions import HPOneViewResourceNotFound
```

Comment:
You forgot to declare  the var HAS_HPE_ONEVIEW in this block (with value equals true)